lib/util/asset_picker_translated_text_delegate.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:localsend_app/gen/strings.g.dart';
+import 'package:wechat_assets_picker/wechat_assets_picker.dart';
+
+
+class TranslatedAssetPickerTextDelegate extends AssetPickerTextDelegate {
+  const TranslatedAssetPickerTextDelegate();
+
+  @override
+  String get languageCode => LocaleSettings.useDeviceLocale().languageCode;
```

Comment:
You can use this: `LocaleSettings.currentLocale.languageCode` to avoid side effects from `useDeviceLocale`

Reply:
Done